packages/pool/contracts/StakeUtils.sol

```diff
@@ -62,10 +62,6 @@ contract StakeUtils is TransferUtils, IStakeUtils {
     /// @notice Called to schedule an unstake by the user
     /// @dev Users need to schedule an unstake and wait for `unstakeWaitPeriod`
     /// to be able to unstake.
-    /// Scheduling an unstake results in the reward of the current epoch to be
-    /// revoked from the user. This is to prevent the user from scheduling
-    /// unstakes that they are not intending to execute (to be used as a
```

Comment:
This is what we are giving up by removing this. This can be largely prevented with a large enough `unstakeWaitPeriod`. For example if it is set to 4 weeks, the user would be able to frontrun claims 25% of the time by having an unstake scheduled at all times.